static/styles/_components/_filters.scss

```diff
@@ -59,12 +68,42 @@
   }
 }
 
+#filters {
+  .accordion__header {
+    border-top: 1px solid $light-gray;
+    padding: 1rem 0;
+  }
+
+  .accordion__item {
+    padding: 0 0 2rem 0;
+    .field {
+      border-top: none;
+      padding: 1rem 0;
+    }
+  }
+}
+
 .any {
   font-weight: normal;
 }
 
+.input--removeable {
+  position: relative;
+}
+
 .button--remove {
   display: none;
   padding: .5rem;
   float: left;
+  position: absolute;  
+  right: 0;
+  top: 0;
+}
+
+.date-range-input {
+  padding-left: 33px;
```

Comment:
I set it in px because it's to align things to a common edge to the right of the radio buttons, which have a fixed width. Still better to use ems or rems? 
![screen shot 2015-07-13 at 3 37 43 pm](https://cloud.githubusercontent.com/assets/1696495/8662502/3bdcd208-2975-11e5-8d2d-e90e4f5de3f4.png)